### PR TITLE
New brightness model + catalog info.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -9,9 +9,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v13
+        with:
+          java-version: adoptium@17
```
[the remark this answers is not in the document]
Comment (rpiaggio) [reply .github/workflows/release.yml:13]: Probably not. Anyway, it's not working for some unclear reason. Let me dial back.

File: modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala [hunk 2]
```diff
@@ -256,58 +270,66 @@ object CatalogAdapter {
...
         findBand(field)
       else
         none
 
-    // Attempts to find the magnitude system for a band
-    override def parseMagnitudeSys(
+    private val integratedBrightnessUnits: Map[String, UnitType @@ Brightness[Integrated]] =
+      Map(
+        "Vega" -> implicitly[IsTaggedUnit[VegaMagnitude, Brightness[Integrated]]],
+        "AB"   -> implicitly[IsTaggedUnit[ABMagnitude, Brightness[Integrated]]]
```
Comment (cquiroz) [inline modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala:310]: What about the other units? Or is that simbad only has AB/Vega?

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala [hunk 3]
```diff
@@ -173,18 +180,29 @@ trait VoTableParser {
...
-        (id, ra, dec, epoch, pv, rv, plx) =>
-          SiderealTracking(CatalogId(adapter.catalog, id).some,
-                           Coordinates(ra, dec),
-                           epoch,
-                           pv,
-                           rv,
-                           plx
+      (parseRA, parseDec, parseEpoch, parsePV, parseRadialVelocity, parsePlx).mapN {
+        (ra, dec, epoch, pv, rv, plx) =>
+          SiderealTracking(
```
Comment (cquiroz) [inline modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:185]: No `CatalogInfo`?

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala [hunk 4]
```diff
@@ -211,11 +229,24 @@ trait VoTableParser {
...
-      }
+    def parseSiderealTarget: ValidatedNec[CatalogProblem, Target.Sidereal] =
+      (parseName, parseSiderealTracking, parseBandBrightnesses, parseCatalogInfo, parseAngularSize)
+        .mapN { (name, pm, mags, info, angSize) =>
+          Target.Sidereal(
+            name,
+            pm,
+            // We set arbitrary values for `sourceProfile`, `spectralDefinition`, `sed` and  `librarySpectrum`: the first in each ADT.
+            // In the future we will attempt to infer some or all of these from the catalog info.
+            SourceProfile.Point(
```
Comment (cquiroz) [inline modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:240]: Does this mean the user will see 05V and will have to select the correct one?

File: modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala (hunk 2 above)
Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala:310]: I based this on the field description for `FLUX_SYSTEM_*` which states `flux system A=AB V=Vega`. But now I see there's another field called `FLUX_UNIT_*` which is probably the one we want to use. However, I cannot find a list of possible values for it.

Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala:310]: Ok, `FLUX_UNIT_*` doesn't seem to be working. According to documentation (http://simbad.u-strasbg.fr/guide/sim-fscript.htx#VotableFields) it should return the flux units but it's returning the name of the band.

I think we should stick with `FLUX_SYSTEM_*` then. Also, despite its description it actually contains values `AB` and `Vega`.

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala (hunk 3 above)
Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:185]: It's now in `Target.Sidereal`.

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala (hunk 4 above)
Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:240]: Yup, they will see `Profile: Point`, `SED: Stellar Library` / `O5V`.

Comment (cquiroz) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:240]: Wouldn't it make more sense that it is empty?

File: modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala (hunk 2 above)
Comment (cquiroz) [reply modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala:310]: 👍

File: modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala (hunk 4 above)
Comment (rpiaggio) [reply modules/catalog/src/main/scala/lucuma/catalog/VoTableParser.scala:240]: In the current model, there's no way to set it as empty.

The profile determines the band brightness units. So, in order to set the band brightness units from Simbad, we need to set a profile that accepts integrated units.

We could still make `SED` optional, but I think we decided on checking the user experience this way and then adjust the model if necessary.

**Pull request merged**